code/modules/clothing/suits/marine_armor/_marine_armor.dm

```diff
@@ -682,3 +682,30 @@
 	icon_state = "cc_armor"
 	flags_atom = NO_SNOW_TYPE|NO_NAME_OVERRIDE
 
+//==================Research Armor==================\\
+
+/obj/item/clothing/suit/storage/marine/caustic
+	name = "\improper XM3-CL Armor"
+	desc = "A lightweight piece of XM3-CL ANCW armor. It has incredibly soft cushioning on the inside, thanks to the gel pads. A truck could hit you and you be fine in this."
+	desc_lore = "An Experimental Model of Anti-Caustic Ligth armormend, or XM-CL, developed against the Xenomporh threat by the USMC and WY. Based on the Model 3 Ligth Armor, this armor system has been modified to allow soldiers to perform unrestricted in battle. It uses CN-Gel to neutalize even the heaviest of xenomoprh acidcs, turning their PH level to 7, wich then glide rigth of the superhydrophobic surface of the armor plates. Its insides are lined with a combination of fiber and gel cushioning, serving as a storage for the CN-Gel, and to protect the user of heavy impacts. CN-Gel also serves as a great insulator, protecting the user from from the harshest of colds, to the blazing sun. Allowing the user to perform in any enviorment. The weigth of the internal systems required the XM3-CL Armor to be striped down tremendously, barley leaving room for any form of protection against projectiles. Making it unsuitable for combat against human combatance. The biological functions of XX-121 revealed to WY and USMC scientist anti caustic propertys in their bodily fluids and allowed them to creat a version of the gel to use xenomorph XX-121 biomatter, primarily their saliva, as a base for a Compound Neutralizing Gel, or CN-Gel. Resulting in a clear, thermoinsulating, thick, and overall economic gel. The Gel itself expands greatly when oxidized though the internal nozzles, allowing a XM3-CL Armor system to be fully operational for weeks before needing to be refilled."
+	icon = 'icons/obj/items/clothing/cm_suits.dmi'
+	icon_state = "caustic_h"
+	item_state = "caustic_h"
+	item_icons = list(
+		WEAR_JACKET = 'icons/mob/humans/onmob/suit_1.dmi'
+	)
+	blood_overlay_type = "armor"
+	flags_atom = parent_type::flags_atom | NO_NAME_OVERRIDE|NO_SNOW_TYPE
```

Comment:
on an unrelated note, are you sure this is necessary?

Reply:
Yes, it needs the NO_NAME_OVERRIDE and NO_SNOW_TYPE thing. Otherwise it will pull a non exsitent sprite if its on a snow map. Says darek.

But i do realise i dont need to define some stuf twice. I will remove the icon defines and then check.

Reply:
I mean the parent_type::flags_atom

Reply:
There are reasons why its not used. Mostly because it has the potential to be hacky which is bad and thats bad and so on. I dont recommend to use it tbh. This should be fine though, probably.

Reply:
nvm its new thats why its not used lol

Reply:
> There are reasons why its not used. Mostly because it has the potential to be hacky which is bad and thats bad and so on. I dont recommend to use it tbh. This should be fine though, probably.

Even i know not to asume things that hard.
Also its used like 4 other times.